### DISCUSSION TITLE
Add `Image` and `Gif` widgets

### DISCUSSION THREAD

**Discussion by ricott1** [ratatui/ratatui#971] [Ideas]
First of all, thanks for this awesome project, I'm really enjoining it :)  

Recently I have been working on some command line games where I display png images and gifs using the `Paragraph` widget. I have some demos on https://github.com/ricott1/sshattrick and https://github.com/ricott1/rebels-in-the-sky.

I wanted to ask if you think there would be place in the repo for an `Image` widget and and maybe a `Gif` stateful widget or if instead they would belong to a separate crate. If so, I could work on them.

Cheers.

**Comment by joshka** [comment]
Definitely they belong in a separate crate. Have you seen the [ratatui-image](https://crates.io/crates/ratatui-image) crate?

**Reply by ricott1** [reply]
nope :) that's pretty cool and supports way more that what I was intending. I'm gonna check it out
